src/views/Home.jsx

```diff
@@ -2,6 +2,7 @@ import './Home.css';
 import { useState } from 'react';
 import { matchToken } from '../api/firebase';
 import { useNavigate } from 'react-router-dom';
+import welcomeImage from '../images/wlecome-img.png';
```

Comment:
**Nitpick (non-blocking):** It looks like we have two image files - `welcome-image.jpeg`, which I think might no longer be used and in that case can be deleted, and `wlecome-img.png` which has a typo in the file name that you might want to fix!